packages/documentation/src/stories/utilities/opacity/opacity.snapshot.stories.ts

```diff
@@ -1,24 +1,10 @@
 import type { Args, StoryContext, StoryObj } from '@storybook/web-components';
 import { html } from 'lit';
-import meta, { opacityOptions } from './opacity.stories';
+import meta, { opacityOptions, Opacity as op } from './opacity.stories';
```

Comment:
If you move the renderer to the meta in `opacity.stories.ts` you can remove this extra import.